ansible_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Ansible", func() {
+	Describe("ansible inventory and execution", func() {
+		Context("when assembling a target list", func() {
+			It("should include an ip and the hostame at least", func() {
+				targets, err := CreateAnsibleTargetsList()
+				hostname, _ = os.Hostname()
+				Expect(err).To(BeNil())
+				Expect(targets).ToNot(BeNil())
```

Comment:
I don't think this can ever be nil, since you're not passing a pointer around. It's always an instantiated array.
Something like:
```suggestion
				Expect(len(targets)).Should(BeNumerically(">", 2))
```